app/messaging/crm/send-crm-message.js

```diff
@@ -0,0 +1,12 @@
+const { crmTopic } = require('../../config')
+const { MessageSender } = require('ffc-messaging')
+const createCrmMessage = require('./create-crm-message')
+
+const sendCrmMessage = async (statement, filename) => {
```

Comment:
Suggest renaming `filename` to `blobUrl` so it's clearer what it is.

Reply:
How did I not change this? copy and paste effect.